test/main.js

```diff
@@ -31,6 +32,48 @@ var options = {
   templateDonut: {
     'twig': './templates/donut.twig'
   },
+  convertCategoryTitles: true,
+  convertCategoryTitlesData: {
+    "categories": {
+      "base": "00-atoms",
```

Comment:
Because `convertCategoryTitles` is true, it will then convert the cat titles. We don't want to open the conversion file for unit tests, so the numbers are hard-coded into the unit-test object